source/includes/api/_subscriptions.md

```diff
@@ -48,6 +48,7 @@ datasetsQuery |              Subscriptions to subscribable datasets
 -- type       | Type of subscription defined in the dataset                       |               Text | Yes (unless `datasets` is specified)
 -- params     | Geographic area of the subscription                               |             Object | Yes (unless `datasets` is specified)
 env           |  Environment of the subscription. Set to `production` by default  |             String | No
+userId        | Id of the user owner of the subscription. This parameter can only be provided when creating subscriptions in requests from other MSs. Otherwise, the id of the user in the token of the request is set as the owner. | String | No
```

Comment:
So, on one side, it's important to have this documented. On the other hand, the fact that a MS can set the userId is not something that concerns an API user, just an API dev. So I'm not sure if this should be here or not, and if it should be elsewhere instead. thoughts?

Reply:
I agree with you on this, definitely worth documenting, but I too was unsure if this was the right place to do it. Maybe the developer docs would be a more suitable place for this, but we have no content there that relates to this (we just have stuff like CT, endpoints, and microservices).

So I'd suggest moving this info to the developer docs, and from here on we document details related to the usage of endpoints by other microservices there. Sounds good?

Reply:
And maybe it would be useful to add a link here to the correct section of the developer docs, to help routing people there!

Reply:
makes sense :+1:

Reply:
Moved this into a new section in the developer docs and added a link here instead.